mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
+exclude = lib/
```

Comment:
Move this to pyproject.toml? :)

Reply:
The setting isn't read at all when I move it to pyproject.toml, not sure if there's a different syntax when the `exclude` is there...

Reply:
But it doesn't! I haven't found any other way to have 'lib' excluded